spec/controllers/rooms_controller_spec.rb

```diff
@@ -90,4 +90,25 @@
       expect(response).to have_http_status(:created)
     end
   end
+
+  describe '#recordings' do
+    it 'ids of recordings that belong only to room with given room_friendly_id' do
+      room1 = create(:room, user:, friendly_id: 'friendly_id_1')
+      room2 = create(:room, user:, friendly_id: 'friendly_id_2')
+      recordings = create_list(:recording, 5, room: room1)
+      create_list(:recording, 5, room: room2)
+      get :recordings, params: { room_friendly_id: room1.friendly_id }
+      recording_ids = JSON.parse(response.body)['data'].map { |recording| recording['id'] }
+      expect(response).to have_http_status(:ok)
+      expect(recording_ids).to eq(recordings.pluck(:id))
```

Comment:
I wouldn't do a direct `eq` here. Look at https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/contain-exactly-matcher

Reply:
I don't think room2 is used?
Is it part of the test?
line 97 and 99

Reply:
```recordings_ids``` seems to have extra brackets for some reason which causes ```contain_exactly``` to fail. Not sure what to use in this case now. We also use ```eq``` above directly in other tests.
![image](https://user-images.githubusercontent.com/38328371/164532186-87e356b9-930f-4054-851c-1b7501de390f.png)

Reply:
`match_array`